robot/impl/local_robot.go

```diff
@@ -1329,23 +1329,34 @@ func (r *localRobot) restartSingleModule(ctx context.Context, mod *config.Module
 		}
 	}
 	diff := config.Diff{
-		Left:     r.Config(),
-		Right:    r.Config(),
-		Added:    &config.Config{},
-		Modified: &config.ModifiedConfigDiff{Modules: []config.Module{*mod}},
-		Removed:  &config.Config{},
+		Left:    r.Config(),
+		Right:   r.Config(),
+		Removed: &config.Config{},
+	}
+	if isRunning {
```

Comment:
Can you leave a comment above this `if-else` to explain why we use modified for a running module and added for a crashed one?

Reply:
done in eca087175